Code/Framework/AzFramework/CMakeLists.txt

```diff
@@ -93,6 +94,8 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
             NAME AZ::AzFramework.Tests
         )
 
+        include(${pal_dir}/platform_${PAL_PLATFORM_NAME_LOWERCASE}.cmake)
+
```

Comment:
Can you add a comment explaining that this is used to add the Xcb Tests Target for supported platforms?

By itself this "include" just looks out there with no context.

Reply:
Maybe it could be more self-explanatory with a different filename? How about `${pal_dir}/platform_specific_test_targets.cmake`?

Reply:
I used the different filename approach.

Reply:
That is good with me.